dipy/nn/histo_resdnn.py

```diff
@@ -69,19 +74,6 @@ def __init__(self, *, sh_order_max=8, basis_type="tournier07", verbose=False):
         verbose : bool (optional)
             Whether to show information about the processing.
             Default: False
-
-        References
-        ----------
-        ..  [1] Nath, V., Schilling, K. G., Parvathaneni, P., Hansen,
-            C. B., Hainline, A. E., Huo, Y., ... & Stepniewska, I. (2019).
-            Deep learning reveals untapped information for local white-matter
-            fiber reconstruction in diffusion-weighted MRI.
-            Magnetic resonance imaging, 62, 220-227.
-        ..  [2] Nath, V., Schilling, K. G., Hansen, C. B., Parvathaneni,
-            P., Hainline, A. E., Bermudez, C., ... & Stępniewska, I. (2019).
-            Deep learning captures more accurate diffusion fiber orientations
-            distributions than constrained spherical deconvolution.
-            arXiv preprint arXiv:1911.07927.
```

Comment:
I did not check but it seems 2 differents papers. why you did not use `Nath2019a` and b?

Reply:
arXiv papers are not reviewed, so I looked for the equivalent peer-reviewed version and added that:
https://github.com/dipy/dipy/blob/master/doc/references.bib#L1807